test/e2e/kubectl/kubectl.go

```diff
@@ -1507,11 +1507,13 @@ metadata:
 			Description: The command 'kubectl version' MUST return the major, minor versions,  GitCommit, etc of the Client and the Server that the kubectl is configured to connect to.
 		*/
 		framework.ConformanceIt("should check is all data is printed ", func() {
-			version := framework.RunKubectlOrDie(ns, "version")
-			requiredItems := []string{"Client Version:", "Server Version:", "Major:", "Minor:", "GitCommit:"}
+			// windows newline to \n
+			versionString := strings.Replace(framework.RunKubectlOrDie(ns, "version"), "\r\n", "\n", 1)
```

Comment:
This should not be needed.

Reply:
I have updated it but LGTM label was removed, could you add it again?